templates/gettingstarted.html

```diff
@@ -43,14 +43,14 @@ <h6>Invenio 3.0</h6>
           <i class="large-icon fa fa-github" aria-hidden="true"></i>
         </div>
         <div class="col-md-9">
-            <p>Invenio is a modular framework of independent collaborative component packages. You can find a <a href="http://invenio.readthedocs.io/en/latest/modules/index.html">list of available packages</a> in our documentation.  The packages are located in three different GitHub organisations:</p>
+            <p>Invenio is a modular framework of independent collaborative component packages. The packages are located in three different GitHub organisations:</p>
```

Comment:
I couldn't find the list of modules in the current documentation. Maybe the bundles page could be used but it needs to be fixed: https://invenio.readthedocs.io/en/latest/developersguide/bundles.html